CHANGELOG.md

```diff
@@ -2,6 +2,112 @@
 
 Changelogs for versions not listed here can be found at https://github.com/DataDog/dd-trace-py/releases
 
+---
+
+## 2.12.0
+
+### New Features
+
+- openai: This introduces 'model' tag for openai integration metrics for consistency with the OpenAI SaaS Integration. It has the same value as <span class="title-ref">openai.request.model</span>- database_clients: Adds `server.address` tag to all `<database>.query` spans (ex: postgres.query). This tag stores the name of the database host.
+- database_clients: Adds `server.address` tag to all `<database>.query` spans (ex: postgres.query). This tag stores the name of the database host.
+- LLM Observability: Flush the buffer of spans to be sent when the payload size would otherwise exceed the payload size limit for the event platform.
```

Comment:
```suggestion
- LLM Observability: Flushes the buffer of spans to be sent when the payload size would otherwise exceed the payload size limit for the event platform.
```